src/utils/date.js

```diff
@@ -23,13 +23,14 @@ export const convertLastUpdateTime = date => {
     return LAST_UPDATE_DATE_ERROR;
   }
 
-  const isValidDate = dateValidator(date);
+  const dateArrWithoutDot = date.split('.');
+  const ISODateString = new Date(dateArrWithoutDot[0]).toISOString();
```

Comment:
it is necessary to add the "T" symbol between the date and time for correct operation